Makefile

```diff
@@ -12,3 +12,16 @@ html:
 PHONY: clean
 clean:
 	rm -rf docs
+
+TOFORMATJUVIXFILES = ./Stdlib
+TOFORMAT = $(shell find ${TOFORMATJUVIXFILES} -name "*.juvix" -print)
+
+.PHONY: $(TOFORMAT)
+juvix-format: $(TOFORMAT)
+$(TOFORMAT): %:
+	@echo "Formatting $@"
+	@juvix dev scope $@ --with-comments > $@.tmp
+	@echo "" >> $@.tmp
+	@mv $@.tmp $@
```

Comment:
Shouldn't we do `rm $@.tmp` afterwards? Otherwise, you're left littered with all the *.tmp files.

Reply:
It's moving the file, not copying it.

Reply:
Sorry, I'm so absent-minded that I saw `cp` instead of `mv`